app/assets/javascripts/statistics.js

```diff
@@ -64,6 +71,18 @@ var getScale = function getScale() {
   return scale
 }
 
+var prohibitStep = function prohibitStep(elementSelector) {
```

Comment:
@dissident I think you may write it simpler using `toggleClass`:
```js
function toggleStep(elementSelector, state) {
  element = $(elementSelector);
  element.find('a').toggleClass('hide', state);
  element.find('span').toggleClass('hide', state);
}
```
And then use
```js
toggleStep('.right_step_wrapper', step == 0)
```